### ISSUE TITLE
problem not using direct_io as mount option

### ISSUE THREAD

**Issue by mokraemer** [immobiliare/sfs#7]
Hi,
I use sfs now in production environment. I found an issue, which can really be bad and I wonder you haven't seen this.
Since rsync synchronizes files on the underlying filesystem (mnt/data) the kernel doesn't recognize the changes & uses the cached data. So
diff /mnt/data/x.txt /mnt/fuse/x.txt
may result in a difference which can cause trouble.

I'm not sure if there is another way to prevent this, since it is not desireable to disable the cache on the filesystem, but reading the false data is neither.

**Comment by lucabrunox** [comment]
@mokraemer true, for our use case it's ok to use an aggressive caching on the fuse layer, because files are not going to be modified, they are either created or deleted. Perhaps instead of `kernel_cache` you can use `auto_cache` or `direct_io`.

The `direct_io` disables cache on the fuse layer, which is bad only for something like sshfs or any other network fuse. Otherwise, if the fuse layer is backed by a local filesystem, the kernel will certainly use the cache for the underlying file system.

Does using `auto_cache` or `direct_io` work for you? In which case we can change the documentation.

**Comment by mokraemer** [comment]
ok. thx. I'll give them a try.
